### PR TITLE
Menu once again

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes

screen shot 2018-06-28 at 17 27 43

### CODE REVIEW

File: administrator/components/com_cpanel/View/System/HtmlView.php [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+				'label'   => 'MOD_MENU_INSTALLER_SUBMENU_DISCOVER',
+				'desc'    => 'MOD_MENU_INSTALLER_SUBMENU_DISCOVER',
+				'icon'    => 'cog'
+			],
+
+			// Updates
+			'com_joomlaupdate' => [
+				'link'    => 'index.php?option=com_installer&view=manage',
+				'enabled' => false,
+				'title'   => 'System update',
```
Comment (brianteeman) [inline administrator/components/com_cpanel/View/System/HtmlView.php]: Why is this one not a language string?

File: administrator/components/com_cpanel/View/System/HtmlView.php [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+			static::$notEmpty                     = true;
+		}
+
+		if (static::$notEmpty)
+		{
+			$this->links = $links;
+		}
+		else
+		{
+			throw new JUserAuthorizationexception(JText::_('JERROR_ALERTNOAUTHOR'), 403);
```
Comment (brianteeman) [inline administrator/components/com_cpanel/View/System/HtmlView.php]: use namespaces eg Text not JText

File: administrator/components/com_cpanel/tmpl/system/default.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * @package     Joomla.Administrator
+ * @subpackage  com_admin
+ *
+ * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+?>
+<h3>System Configuration</h3>
```
Comment (brianteeman) [inline administrator/components/com_cpanel/tmpl/system/default.php]: Why isnt this a language string

File: administrator/components/com_cpanel/tmpl/help/default.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+use Joomla\CMS\Language\Text;
+?>
+<h3><?php echo Text::_('MOD_MENU_HELP'); ?></h3>
+<div class="card-columns">
+
+	<?php foreach ($this->links as $link) : ?>
+		<div class="card">
+			<div class="card-block">
+				<h4 class="card-title"><?php echo Text::_($link['label']); ?></h4>
+				<span class="fa fa-<?php echo $link['icon']; ?> fa-5x"></span>
```
Comment (brianteeman) [inline administrator/components/com_cpanel/tmpl/help/default.php]: accessibility

File: administrator/components/com_cpanel/tmpl/system/default.php [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+<div class="card-columns">
+	<?php foreach ($this->links as $name => $link) : ?>
+	<?php if ($link['enabled']) : ?>
+		<div class="card">
+			<div class="card-block">
+				<h4 class="card-title"><?php echo JText::_($link['label']); ?></h4>
+				<span class="fa fa-<?php echo $link['icon']; ?> fa-5x"></span>
```
Comment (brianteeman) [inline administrator/components/com_cpanel/tmpl/system/default.php]: accessibility

**Comment by dgrammatiko** [comment]
@brianteeman thanks for the comments, this PR is just the resurrection of https://github.com/joomla/joomla-cms/pull/15660 and it will be the base for some coordinated work to actually make the new menu adhere to the approved design of Lucas and Elisa. Your more than welcome to be part of this effort but before trying to edit any html wait for @ciar4n to do the initial styling here

**Pull request merged**